packages/aws-cdk-lib/cx-api/lib/features.ts

```diff
@@ -104,6 +104,7 @@ export const KMS_REDUCE_CROSS_ACCOUNT_REGION_POLICY_SCOPE = '@aws-cdk/aws-kms:re
 export const PIPELINE_REDUCE_ASSET_ROLE_TRUST_SCOPE = '@aws-cdk/pipelines:reduceAssetRoleTrustScope';
 export const EKS_NODEGROUP_NAME = '@aws-cdk/aws-eks:nodegroupNameAttribute';
 export const EBS_DEFAULT_GP3 = '@aws-cdk/aws-ec2:ebsDefaultGp3Volume';
+export const REMOVE_DEFAULT_DEPLOYMENT_ALARM = '@aws-cdk/aws-ecs:removeDefaultDeploymentAlarm';
```

Comment:
should we rename it to  `REMOVE_DEFAULT_ECS_DEPLOYMENT_ALARM` to make it more clear which service is it associated to?

Reply:
Good call, let me update it.